site/content/docs/user/quick-start.md

```diff
@@ -425,6 +425,50 @@ The structure of the logs will look more or less like this:
 The logs contain information about the Docker host, the containers running 
 kind, the Kubernetes cluster itself, etc.
 
+### Rootless Docker
```

Comment:
I think this should maybe be a separate guide in the docs similar to WSL2, this page is a bit lengthy already.

Reply:
updated